pkgs/top-level/perl-packages.nix

```diff
@@ -18842,6 +18842,13 @@ with self; {
     };
     buildInputs = [ TestDeep TestDifferences TestException TestMost TestWarn ];
     propagatedBuildInputs = [ CDDBFile Moo ];
+    patches = [
+      # https://rt.cpan.org/Public/Bug/Display.html?id=152242
+      (fetchpatch2 {
+        url = "https://sources.debian.org/data/main/libn/libnet-freedb-perl/0.10-5/debian/patches/implicit-function-declaration.patch";
+        sha256 = "0lp7brxck5azlcsf3hir599cajygvx57vvqvjgcbadg7jk17y1fk";
```

Comment:
```suggestion
        hash = "sha256-oIfJuLzTm0JLzAUvP1mcGCm6oWGpcLWDPqHN/4pKC9Q=";
```

Reply:
Thanks. I just rebased and made the suggested update.